python/cudf/cudf/_lib/orc.pyx

```diff
@@ -249,7 +249,7 @@ cdef orc_reader_options make_orc_reader_options(
         .timestamp_type(data_type(timestamp_type))
         .use_index(use_index)
         .decimal_cols_as_float(c_decimal_cols_as_float)
-        .decimal128(False)
+        .decimal128(True)
```

Comment:
This line can be removed. Will remove the API in the next release.
```suggestion
```

Reply:
Removed, Thanks @vuule